Rubberduck.Core/UI/Converters/BoolToVisibilityConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace Rubberduck.UI.Converters
+{
+    class BoolToVisibilityConverter : IValueConverter
```

Comment:
This class is already built into the framework. 

https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.booleantovisibilityconverter?view=netframework-4.8